tests/testdata/programs/replace-cluster-add-subnets/index.ts

```diff
@@ -22,7 +22,16 @@ const popped = pulumi.output(publicSubnetIds).apply(subnets => {
 const cluster = new eks.Cluster(`${projectName}`, {
     vpcId: vpc.vpcId,
     publicSubnetIds: popped,
-});
+}, { transforms: [args => {
```

Comment:
This test was not working correctly anymore. Previously changes to the subnets triggered replacements. this is not the case anymore. This test wants to test a cluster replacement so I forced it.

Background why this wasn't failing hard in the past:
EKS picks two of the provided subnets for its control plane ENIs. When updating them, they need to be in the same AZs as the original ones.
We have a 1/3 chance to pick the right ones (the VPC has 3 AZs) and we have 3 retries configured for the test. So we got lucky most of the times.